docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -138,6 +138,8 @@ In addition some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+The constructor WrapperQueryBuilder(byte[] source, int offset, int length) has been removed, use WrapperQueryBuilder(byte[] source) instead.
+
```

Comment:
we need to add here the signature change to `public static WrapperQueryBuilder wrapperQuery(byte[] source)` in QueryBuiders.